source/server/admin/stats_render.cc

```diff
@@ -53,8 +53,8 @@ void StatsTextRender::addDetail(const std::vector<Stats::ParentHistogram::Bucket
                                 Buffer::Instance& response) {
   absl::string_view delim = "";
   for (const Stats::ParentHistogram::Bucket& bucket : buckets) {
-    response.addFragments(
-        {delim, absl::StrCat(bucket.lower_bound_, ",", bucket.width_, ":", bucket.count_)});
+    response.addFragments({delim, absl::StrFormat("%.15g,%.15g:%lu", bucket.lower_bound_,
```

Comment:
Just making sure I understand: this change increases the precision of the formatting, correct?

Why doesn't this one use the new API?

Reply:
comments crossed paths...see above.

Reply:
and yes, it increases precision relative to StrCat.